sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -399,6 +415,18 @@ export { PollOperationState }
 export interface PurgeDeletedKeyOptions extends coreHttp.OperationOptions {
 }
 
+// @public
+export interface ReleaseKeyOptions extends coreHttp.OperationOptions {
+    algorithm?: KeyExportEncryptionAlgorithm;
+    nonce?: string;
```

Comment:
nonce is optional, and it works fine without one. I do want to check with the service team if there's value in providing one if the customer does not pass one in, but for now I left it as is

Reply:
To avoid tight coupling, I recommend letting the service define it if not provided. It also makes the code more portable since not every environment provides a cryptographic RNG.